regression/CMakeLists.txt

```diff
@@ -14,17 +14,20 @@ function(add_esbmc_regression folder mode)
 endfunction(add_esbmc_regression mode folder)
 
 if(APPLE)
-    set(REGRESSIONS esbmc cbmc cstd llvm floats floats-regression k-induction esbmc-unix esbmc-unix2 k-induction-parallel)
+    set(REGRESSIONS esbmc cbmc cstd llvm floats floats-regression k-induction esbmc-unix esbmc-unix2 k-induction-parallel nonz3)
+elseif(WIN32)
+    # FUTURE: Add floats-regression esbmc-cpp/cpp
+    set(REGRESSIONS esbmc cbmc cstd llvm floats  k-induction )
```

Comment:
Why isn't the floats-regression included here?

Reply:
because the models for it are not finished yet. I intend to open an issue for this

Reply:
I see, like the signbit one? I agree, they should come in a separate PR.